### PR TITLE
add mlx-image

### PR DESCRIPTION
**Pull request by riccardomusmeci**

### CODE REVIEW

File: packages/tasks/src/model-libraries-snippets.ts [hunk 1]
```diff
@@ -532,6 +532,13 @@ export HF_HUB_ENABLE_HF_TRANS: string[]FER=1
 huggingface-cli download --local-dir ${nameWithoutNamespace(model.id)} ${model.id}`,
 ];
 
+export const mlxim = (model: ModelData): string[] => [
+	`pip install mlx-image
```
Comment (merveenoyan) [inline packages/tasks/src/model-libraries-snippets.ts:536]: No need for installation part :)

**Review by merveenoyan — COMMENTED** [review]
minor nit :)

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -192,6 +192,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.mlx,
 		filter: true,
 	},
+	"mlxim": {
```
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:195]:
```suggestion
	mlxim: {
```

Linter will complain otherwise :)

**Review by pcuenca — COMMENTED** [review]
LGTM. Not to be merged before https://github.com/huggingface/hub-docs/pull/1237/files

File: packages/tasks/src/model-libraries.ts [hunk 3]
```diff
@@ -192,6 +192,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
...
 		filter: true,
 	},
+	mlxim: {
+		prettyLabel: "mlxim",
+		repoName: "mlx-image",
+		repoUrl: "https://github.com/riccardomusmeci/mlx-image",
+		docsUrl: "https://huggingface.co/docs/hub/mlxim",
+		snippets: snippets.mlxim,
+		filter: false,
+		countDownloads: { term: { path: "model.safetensors" } }
```
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:202]:
```suggestion
		countDownloads: { term: { path: "model.safetensors" } },
```

**Pull request merged**